node/src/components/consensus/highway_core/state/tests.rs

```diff
@@ -384,21 +385,21 @@ fn validate_lnc_one_equivocator() -> Result<(), AddUnitError<TestContext>> {
         UnitError::LncNaiveCitation(ALICE)
     );
     endorse!(state, rng, CAROL, c0);
+    // TODO: Test missing endorsement dependencies.
     // One endorsement isn't enough.
-    assert_eq!(
-        add_unit!(state, rng, DAN, None; F, b0, c0, N; c0)
-            .unwrap_err()
-            .cause,
-        UnitError::LncNaiveCitation(ALICE)
-    );
+    // assert_eq!(
+    //     add_unit!(state, rng, DAN, None; F, b0, c0, N; c0)
+    //         .unwrap_err()
+    //         .cause,
+    //     UnitError::LncNaiveCitation(ALICE)
+    // );
```

Comment:
Should we remove this bit entirely then?

Reply:
Maybe, yes.
I think the dependency stuff is relatively simple code, compared to the actual LNC.

Reply:
Ok, I'm in favor then. If we use `add_unit!` macro then it's not a place for it. We _could_ revert this particular unit test to work with `WireUnit { … }` and `state.validate_lnc` directly again.